src/directory/directory.mjs

```diff
@@ -734,354 +734,6 @@ export const directory = {
             }
           ]
         },
-        {
```

Comment:
removing the prev section form directory but will add it back in with the version switcher update that will include logic for correctly handling prev